spec/features/movie_index_page_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+feature "view list of movies on movie index page", %Q{
+    As any user
+    I want to visit the index page
+    So that I can see all of the movie titles
+  }  do
+
+    let!(:movie1) {Movie.create(movie_title: "Test Title 1", director: "Test Director 1", year: "2022", description: "Not as good as the book")}
+    let!(:movie2) {Movie.create(movie_title: "Test Title 2", director: "Test Director 2", year: "2023")}
+    let!(:movie3) {Movie.create(movie_title: "Test Title 3", director: "Test Director 1", year: "2024")}
+
+    scenario "visit homepage" do
+
+        visit movies_path
+
+        expect(page).to have_content "Test Title 1"
+        expect(page).to have_content "Test Title 2"
+        expect(page).to have_content "Test Title 3"
+
```

Comment:
this file looks to have a tab indentation size of 4, whereas other files have a tab size of 2. This should be consistent across group members! I'd recommend adhering to a tab size of 2 and can be updated in your VSCode settings